eoscostarica/eos-rate
# design(logo) Update logo version

### GH Issue

# Update EOS Costa Rica Logo #232

### Steps to test

1. Run webapp locally
1. Check logo in footer
1. Check logo in readme file

### Checklist
